onnxruntime/test/python/quantization/test_save_activations.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""Testing save_activations module
+"""
```

Comment:
```suggestion
"""Tests for the save_activations module."""
```

super nit: formatting